site/src/pages/components/invokers.explainer.mdx

```diff
@@ -357,6 +357,119 @@ will be `aria-expanded=true` when the _Invokee_ is open and
 
 TBD: Accessibility attributes for `interesttarget`.
 
+### Security
+
+This proposal adds new capabilities in that elements can be interacted with
+using HTML where prior these required scripting (read: JS). Care has been taken
+within this proposal to ensure there are no substantially new security
+implications, and that the risk is kept low.
+
+In general, _preventing scripting is not a security barrier_ because very few
+users or websites disable scripting. That being said, we should not grant
+significant new capabilities with or without scripting.
+
+Invokers, by design, have two security affordances:
+
+- They require user interaction (invoke events and their default behaviours can
+  only happen with a trusted click), so attackers are unable to synthesise
+  click events via scripting in order to trigger the invoke behaviour.
+- The `invoketarget` takes an IDREF which must reference an ID within the same
+  document. `invokeTargetElement` can be assigned a cross-root element, but
+  this requires scripting which means all actions are already possible.
+
+To go into more detail, below is a brief summary of the security considerations
+per interaction:
+
+#### Popovers
+
+This proposal allows opening and closing popovers with `invoketarget`. This is
+already possible with `popovertarget`, and so there is no new capability here.
+
+#### Details
+
+This proposal allows opening and closing `<details>` elements with
+`invoketarget`. This is not considered to be new capability, as it is already
+possible to do this with the `<summary>` element.
+
+#### Input
+
+This proposal allows showing the pickers of input elements, for example an
+invoker can target an `<input type=file>` and clicking the invoker will open
+the operating systems file picker dialog. This is not new capability, as this is
+currently possible by crafting a `<label>` element that points to or wraps the
+input, hiding the input and styling the label to look like a button. Interacting
+with (clicking) the label will invoke the input picker.
+
+It should be noted that input pickers (like the file picker dialog) render native
+controls outside of the bounds of the document frame (iframe or browser window).
+Consequently these behaviours shoild not run in cross origin iframe contexts,
+which is the case today calling the `.showPicker()` API on those elements.
```

Comment:
Technically today .showPicker works cross-origin for file and color inputs but not the others. This proposal does not have that special casing. It might be worth calling this out?